controller/src/main/java/org/jboss/as/controller/descriptions/DefaultOperationDescriptionProvider.java

```diff
@@ -126,9 +130,22 @@ public DefaultOperationDescriptionProvider(final String operationName,
         this.deprecationData = deprecationData;
         this.replyParameters = replyParameters;
         this.accessConstraints = accessConstraints != null ? accessConstraints : Collections.<AccessConstraintDefinition>emptyList();
+        this.stability = Stability.DEFAULT;
```

Comment:
> I think this could be suspicious, this description provider is somehow tight to a specific operation definition; the constructor is using an operation name. An Operation definition is Feature aware, since it implements it.

Frankly, I would normally have deprecated this constructor (and the others).  However, there seems to be a least use of this constructor in org.wildfly.core:wildfly-host-controller, but its usage seem dubious.  I'm in the process of tracking down the author to verify.

> However, in this constructor, the Operation definition feature stream is unknown because it uses an Operation name. I am wondering if that could lead to problems where an Operation running on a different stability level above the default one, could be using this constructor.

The same could be said for any aspect of the operation description.  Using any of the existing constructors, a user could describe their operation in a manner inconsistent with the OperationDefinition.